benchmark/src/main/scala/fs2/benchmark/StreamBenchmark.scala

```diff
@@ -2,20 +2,29 @@ package fs2
 package benchmark
 
 import cats.effect.IO
-import org.openjdk.jmh.annotations.{Benchmark, BenchmarkMode, Mode, OutputTimeUnit, Scope, State}
+import org.openjdk.jmh.annotations.{
+  Benchmark,
+  BenchmarkMode,
+  Mode,
+  OutputTimeUnit,
+  Param,
+  Scope,
+  State
+}
 import java.util.concurrent.TimeUnit
 
 @State(Scope.Thread)
 class StreamBenchmark {
-  @GenerateN(1, 4096, 665536, 67108863)
+  @Param(Array("10", "100", "1000", "10000"))
```

Comment:
this array (10, 100, 1000...) is different than the old.  Looks like the old was on specific boundaries, i wonder if there was a specific test in mind.

Certainly, 10,000 will be different than 67,108,863 as a max upper bounds?

Reply:
Good catch, fixed